### PR TITLE
III-6082 Upgrade phpstan

### PR DESCRIPTION
**Pull request by grubolsch**
### Changed
Upgrade phpstan to a higher version

---
Ticket: https://jira.publiq.be/browse/III-6082

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -99,3 +99,32 @@ jobs:
 
         - name: 🔍 Run static analysis
           run: composer phpstan
+
+    phpunit:
```
Comment (LucWollants) [inline .github/workflows/ci.yml:103]: Why is this added? Is this not already here https://github.com/cultuurnet/jwt-provider/pull/124/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR6 ?

**Comment by LucWollants** [comment]
Looks like a lot of packages got updated. Pity that we lack acceptance tests, did you try it locally @grubolsch ?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (grubolsch) [reply .github/workflows/ci.yml]: You are right, I removed it again

**Review by simon-debruijn — APPROVED** [review]
